php-checks/src/test/resources/checks/security/XxeCheck.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+function simpleXML() {
+  $xml = file_get_contents("xxe.xml");
+  $doc = simplexml_load_string($xml, "SimpleXMLElement");
+  $doc = simplexml_load_string($xml, "SimpleXMLElement", 42);
+  $doc = simplexml_load_string($xml, "SimpleXMLElement", LIBXML_NOENT); // Noncompliant
+//                                                       ^^^^^^^^^^^^
+  $doc = simplexml_load_string($xml, "SimpleXMLElement", LIBXML_NOBLANKS);
+  $doc = simplexml_load_string($xml, 'SimpleXMLElement', LIBXML_NOBLANKS | LIBXML_NOENT); // Noncompliant
+//                                                                         ^^^^^^^^^^^^
+  $doc = simplexml_load_string($xml, 'SimpleXMLElement', LIBXML_NOENT | LIBXML_NOBLANKS); // Noncompliant
+  $doc = simplexml_load_string($xml, 'SimpleXMLElement', LIBXML_COMPACT | LIBXML_NOBLANKS);
+  $doc = simplexml_load_string($xml, 'SimpleXMLElement', LIBXML_COMPACT | (LIBXML_NOBLANKS | LIBXML_NOENT)); // Noncompliant
+  $doc = simplexml_load_string($xml, LIBXML_NOENT);
+  $doc = simplexml_load_string($xml, options: LIBXML_NOENT); // Noncompliant
+  $doc = $obj->simplexml_load_string($xml, "SimpleXMLElement", LIBXML_NOENT);
+  $doc = unrelated_function($xml, "SimpleXMLElement", LIBXML_NOENT);
+
+  $options1 = LIBXML_NOBLANKS | LIBXML_NOENT; // Noncompliant
```

Comment:
shouldn't be line 21 to be non compliant?

Reply:
I was not sure about that one, but I think you're right.
Changed in edc54169b.